jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -138,6 +142,43 @@ ecma_op_object_get_own_property (ecma_object_t *object_p, /**< the object */
       }
       break;
     }
+#ifndef CONFIG_DISABLE_TYPEDARRAY_BUILTIN
+    case ECMA_OBJECT_TYPE_PSEUDO_ARRAY:
+    {
+      /* ES2015 9.4.5.1 */
+      if (ecma_is_typedarray (ecma_make_object_value (object_p)))
+      {
+        if (ECMA_STRING_GET_CONTAINER (property_name_p) == ECMA_STRING_CONTAINER_UINT32_IN_DESC)
+        {
+          ecma_value_t value = ecma_op_typedarray_get_index_prop (object_p, property_name_p->u.uint32_number);
+          if (!ecma_is_value_undefined (value))
+          {
+            property_ref_p->virtual_value = value;
```

Comment:
Virtual value only needed to return if ECMA_PROPERTY_GET_VALUE is specified. Perhaps ecma_op_typedarray_get_index_prop could be more clever to return just a TRUE simple value or something if the value is not needed to get.